@here/harp-examples/src/object-picking.ts

```diff
@@ -221,7 +221,8 @@ export namespace PickingExample {
 
         const omvDataSource = new OmvDataSource({
             baseUrl: "https://vector.hereapi.com/v2/vectortiles/base/mc",
-            authenticationCode: apikey
+            authenticationCode: apikey,
+            gatherFeatureAttributes: true
```

Comment:
this was accidentally removed from the previous PR https://github.com/heremaps/harp.gl/pull/1591